Source/MultipartFormData.swift

```diff
@@ -612,18 +582,18 @@ public class MultipartFormData {
 
     // MARK: - Private - Content Headers
 
-    private func contentHeaders(name: String) -> [String: String] {
+    private func contentHeaders(withName name: String) -> [String: String] {
         return ["Content-Disposition": "form-data; name=\"\(name)\""]
     }
 
-    private func contentHeaders(name: String, mimeType: String) -> [String: String] {
+    private func contentHeaders(withName name: String, mimeType: String) -> [String: String] {
         return [
             "Content-Disposition": "form-data; name=\"\(name)\"",
```

Comment:
Or perhaps there's a way to refactor this group of related functions into fewer?